site/_posts/2016-11-07-last-week-in-manageiq-bugs-and-npm.html.md

```diff
@@ -0,0 +1,68 @@
+---
+title: Last week in ManageIQ - Bugs and NPM
+author: himdel
+date: 2016-11-07
+comments: true
+published: true
+tags: LWIMIQ
+---
+
+[Martin](https://github.com/himdel) here, bringing you this week's edition of Last Week in ManageIQ.
+
+
+## Featured
+
+Since the last edition, we've merged 114 [pull requests](https://github.com/ManageIQ/manageiq/pulls?utf8=%E2%9C%93&q=is%3Apr%20is%3Amerged%20base%3Amaster%20merged%3A%222016-10-29%20..%202016-11-06%22%20sort%3Acreated-desc%20) in the `manageiq` repo, 56 of which were labeled `bug`, 18 `enhancement`, 16 `technical debt` and 13 `refactoring`. As always, almost half was in the UI, totalling 48 `ui` pull requests.
+
+In the recently renamed `manageiq-ui-service` repo, we've had 8 [pull requests](https://github.com/ManageIQ/manageiq-ui-service/pulls?utf8=%E2%9C%93&q=is%3Apr%20is%3Amerged%20base%3Amaster%20merged%3A%222016-10-29%20..%202016-11-06%22%20sort%3Acreated-desc%20), and there's surely more in our other repos.
+
+There's also many [open issues] you can try your skills on, because who doesn't want to be [famous](https://github.com/ManageIQ/manageiq/graphs/contributors)? And, as always, we're ready to help on [gitter].
+
+
+## Improved
+
+As a sign of things to come, [Allen](https://github.com/AllenBW) has finished moving our Service UI from `bower` to `npm` (actually, `yarn`, we're so modern!) in [#296](https://github.com/ManageIQ/manageiq-ui-service/pull/296), ManageIQ itself will follow shortly.
```

Comment:
Tiny nit - but the tools are Bower, npm and Yarn. Their binaries will be `bower`, etc..